app/src/main/res/values-de/strings.xml

```diff
@@ -166,9 +166,9 @@
     <string name="nc_permissions_settings">Einstellungen öffnen</string>
 
     <!-- Call -->
-    <string name="nc_voice_call">Nextcloud Talk Sprachanruf</string>
```

Comment:
This will be overwritten again via Transifex, so please do not commit this.
Best practice is not to change an already existing string, but use a new one.
Otherwise you might have strange UX if a language is not updated.
With having a new string, worst what could happen is that it is English…

Reply:
ah sure. renamed the keys and changed the values